lib/improver/blending/weighted_blend.py

```diff
@@ -125,17 +239,31 @@ def conform_metadata(
             ndim = cube.coord_dims("time")
             cube.add_aux_coord(forecast_period, data_dims=ndim)
 
+    # update blended cube attributes
+    if "title" not in cube.attributes.keys():
+        cube.attributes["title"] = "IMPROVER Model Forecast"
+
+    if ("model" in coord) and any(
+            "mosg__" in key for key in cube_orig.attributes.keys()):
+        cube.attributes["mosg__model_configuration"] = "blend"
+
+    # inherit grid attributes
+    for key in cube_orig.attributes.keys():
+        if "mosg__grid" in key:
+            cube.attributes[key] = cube_orig.attributes[key]
+
+    # remove appropriate scalar coordinates
     for coord in ["model_id", "model_realization", "realization"]:
         if cube.coords(coord) and cube.coord(coord).shape == (1,):
             cube.remove_coord(coord)
 
-    if coords_for_bounds_removal is None:
-        coords_for_bounds_removal = []
+    # remove bounds from specified scalar coordinates
```

Comment:
Do we need to keep this functionality now the interface to it has gone from the CLI?

Reply:
I wasn't sure, so didn't want to remove it in case there was another use case.  Will ask.